### PR TITLE
Copy CONTRIBUTING.md from skopeo

### PR DESCRIPTION
**Pull request by rhatdan**
We need to tell people about contributing to the CRI-O project

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes: https://github.com/kubernetes-incubator/cri-o/issues/1071

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+No Pull Request (PR) is too small! Typos, additional comments in the code,
+new testcases, bug fixes, new features, more documentation, ... it's all
+welcome!
+
+While bug fixes can first be identified via an "issue", that is not required.
+It's ok to just open up a PR with the fix, but make sure you include the same
+information you would have included in an issue - like how to reproduce it.
+
+PRs for new features should include some background on what use cases the
+new code is trying to address. And, when possible and it makes, try to break-up
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: "And, when possible and it makes," to "When possible and when it makes sense,"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+larger PRs into smaller ones - it's easier to review smaller
+code changes. But, only if those smaller ones make sense as stand-alone PRs.
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: suggested: remove comma after "But"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+Regardless of the type of PR, all PRs should include:
+* well documented code changes
+* additional testcases. Ideally, they should fail w/o your code change applied
+* documentation changes
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: Do you want to add something about signing the commit here?  Seems to be a forgotten quite a bit lately.

Comment (TomSweeneyRedHat) [reply CONTRIBUTING.md]: nvm, I see it below.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    Signed-off-by: Joe Smith <joe.smith@email.com>
+
+Use your real name (sorry, no pseudonyms or anonymous contributions.)
+
+If you set your `user.name` and `user.email` git configs, you can sign your
+commit automatically with `git commit -s`.
+
+## Communications
+
+For general questions, or dicsussions, please use the
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: "dicsussions" to "discussions"

**Comment by TomSweeneyRedHat** [comment]
@rhatdan excellent start, just about there after a few nits to pick.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Added your fixes, we should probably correct in skopeo, and I will add this page to buildah.

**Comment by rhatdan** [comment]
oops I had the wrong paths to the git repo,  Fixed

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+Use your real name (sorry, no pseudonyms or anonymous contributions.)
+
+If you set your `user.name` and `user.email` git configs, you can sign your
+commit automatically with `git commit -s`.
+
+## Communications
+
+For general questions, or discussions, please use the
+IRC group on `irc.freenode.net` called `container-projects`
```
Comment (runcom) [inline CONTRIBUTING.md]: this should say `cri-o` channel on freenode

**Comment by runcom** [comment]
Except one nit, LGTM

**Comment by rhatdan** [comment]
Merging.

**Pull request merged**